src/sass/modules/_m-vet-nav.scss

```diff
@@ -1,5 +1,5 @@
 // The distance from the top of the sceen that the mobile navigation menu should appear
-$height-offset: 111.16px;
+$height-offset: 121.16px;
 
```

Comment:
This value was increased because I removed the margin properties from the menu. The margin was causing weird scroll issues on mobile where some menus would be cut off by the value of the margin.